bluemira/equilibria/equilibrium.py

```diff
@@ -168,7 +159,22 @@ def get_forces(self):
         F: np.array(n_coils, 2)
             [Fx, Fz] array of forces on coils [MN]
         """
-        return self.force_field.calc_force(self.coilset.get_control_currents())[0] / 1e6
+        coils = list(self.coilset.coils.values())
+        currents = self.coilset.get_control_currents()
+        plasma = self.plasma_coil()
+        response = np.zeros((len(coils), len(coils), 2))
+        background = np.zeros((len(coils), 2))
+        for i, coil1 in enumerate(coils):
+            for j, coil2 in enumerate(coils):
+                response[i, j, :] = coil1.control_F(coil2)
+            if coil1.current != 0.0:
+                background[i, :] = coil1.F(plasma) / coil1.current
```

Comment:
Yeah the peak poloidal fields tend to be in the solenoid and these are usually at the mid inboard edge. The coil bulk forces are best estimated at the centroid.

Reply:
Makes complete sense - I'd add this (a mention of the exact point the forces are being evaluated) into the docstring for clarity, as has been done for `get_fields` below, in case there's any desire down the line to evaluate representative forces at any other point instead.